app/code/Magento/Inventory/Model/ResourceModel/PredefinedId.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\ResourceModel;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Model\AbstractModel;
+
+/**
+ * Provide possibility of save entity with predefined id
+ * Use trait instead of extending for better reusability, also we don'nt want to hardcode hierarchy of inheritance
+ */
+trait PredefinedId
+{
+    /**
+     * Check if object is new
```

Comment:
overwrite default AbstractModel implementation of the function isObjectNew
`@see origina` 

adding the possibility to check whether (record) already exists in DataBase or not